src/components/tests/SourcesTree.spec.js

```diff
@@ -41,48 +626,22 @@ function render(overrides = {}) {
   return { component, props };
```

Comment:
i mentioned in an earlier review that since instance is being used so often, you can return it from the render function. 

you can do that here:

```
function render(overrides = {}) {
  const props = generateDefaults(overrides);
  const component = shallow(<SourcesTree.WrappedComponent {...props} />);
  const instance = component.instance;
  return { component, props, instance };
}
```